domain/src/main/kotlin/no/nav/su/se/bakover/domain/regulering/ReguleringRepo.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.su.se.bakover.domain.regulering
+
+import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.domain.Saksnummer
+import java.time.LocalDate
+import java.util.UUID
+
+data class VedtakSomKanReguleres(
```

Comment:
Navnet her er jo OK, men tenkte litt på om det kunne vært enda tydeligere at dette vil være "en del av en periode", hvor man f.eks. for en sak hvor det ligger inne en iverksatt søknadsbehandling, en revurdering og et opphør, vil ende opp i tre `VedtakSomKanReguleres`. I tillegg inneholder den jo en samling av relevant info, ikke bare det samme som et vedtak. 

Ville det f.eks. vært bedre med `RegulerbarPeriode`?